pkg/controller/migration/namespace_migration.go

```diff
@@ -357,6 +364,16 @@ func (m *CoreNamespaceMigration) CleanupMigration(ctx context.Context) error {
 	return nil
 }
 
+// deleteKubeSystemServiceEndPointCm deletes the kubernetes-services-endpoint configmap
+// in the kube-system namespace
+func (m *CoreNamespaceMigration) deleteKubeSystemServiceEndPointCm(ctx context.Context) error {
```

Comment:
"Cm" is short for configMap, so should be same-case given golang conventions (e.g., EndpointCM instead of EndpointCm)

Alternatively, just spell it out - it's already a long function name and ConfigMap is more clear than CM